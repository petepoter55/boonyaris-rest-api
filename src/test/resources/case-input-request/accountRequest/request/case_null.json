{
  "username": null,
  "password": null,
  "confirmPassword": null,
  "role": null,
  "firstname": null,
  "lastname": null,
  "email": null,
  "confirmEmail": null,
  "createBy": "usperadmin",
  "delFlag": true
}