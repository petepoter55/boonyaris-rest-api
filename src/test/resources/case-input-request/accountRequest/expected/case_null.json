[
  "username: must not be null",
  "password: must not be null",
  "confirmPassword: must not be null",
  "role: must not be null",
  "firstname: must not be null",
  "lastname: must not be null",
  "email: must not be null",
  "confirmEmail: must not be null"
]